x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/available_packages.tsx

```diff
@@ -340,7 +340,7 @@ export const AvailablePackages: React.FC<{
               })}
               description={i18n.translate('xpack.fleet.featuredObsDesc', {
                 defaultMessage:
-                  'Monitor, detect and diagnose complex performance issues from your application.',
+                  'Monitor, detect and diagnose complex application performance issues',
```

Comment:
Thanks, @ak20102763!

I just left a comment in the original issue: https://github.com/elastic/apm-server/issues/7635#issuecomment-1181168532. Can we update this to the following:

```suggestion
                  'Monitor, detect, and diagnose complex application performance issues.',
```

Reply:
And can you please sign the CLA via the link in this comment: https://github.com/elastic/kibana/pull/136121#issuecomment-1180607722. Thanks!